src/VirtoCommerce.DescriptionExportImportModule.Data/Services/DescriptionDataExporter.cs

```diff
@@ -0,0 +1,113 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using VirtoCommerce.DescriptionExportImportModule.Core;
+using VirtoCommerce.DescriptionExportImportModule.Core.Models;
+using VirtoCommerce.DescriptionExportImportModule.Core.Services;
+using VirtoCommerce.Platform.Core;
+using VirtoCommerce.Platform.Core.Assets;
+using VirtoCommerce.Platform.Core.Common;
+using VirtoCommerce.Platform.Core.Exceptions;
+using VirtoCommerce.Platform.Core.Extensions;
+
+namespace VirtoCommerce.DescriptionExportImportModule.Data.Services
+{
+    public class DescriptionDataExporter : IDescriptionDataExporter
```

Comment:
Why naming does not match the domain model's name? I mean EditorialReview or just Review.